test/test_array.py

```diff
@@ -24,10 +24,10 @@
 import pytest
 import numpy as np
 
-from arraycontext import _acf           # noqa: F401
-from arraycontext import (              # noqa: F401
-        pytest_generate_tests_for_pyopencl_array_context
-        as pytest_generate_tests)
+from meshmode.array_context import PytestPyOpenCLArrayContextFactory
```

Comment:
```suggestion
from meshmode.array_context import PytestPyOpenCLArrayContextFactory, _acf
```
? Or update the one in `meshmode._acf`?

Reply:
130745984ef3eb193df0128dedd250ef07d35160